src/qt/bitcoin.cpp

```diff
@@ -510,7 +510,11 @@ int GuiMain(int argc, char* argv[])
     // Show help message immediately after parsing command-line options (for "-lang") and setting locale,
     // but before showing splash screen.
     if (HelpRequested(gArgs) || gArgs.IsArgSet("-version")) {
-        HelpMessageDialog help(nullptr, gArgs.IsArgSet("-version"));
+        bool wallet_enabled{false};
```

Comment:
nit: welp, we could go full const by using complete set of `#ifdef / #else / #endif`:

```
#ifdef ENABLE_WALLET
        const bool wallet_enabled{WalletModel::isWalletEnabled()};
#else
        constexpr bool wallet_enabled{false};
#endif // ENABLE_WALLET
```

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/227#issuecomment-786746698).